maintenance_stock/views/maintenance_equipment_views.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="hr_equipment_view_search" model="ir.ui.view">
+        <field name="model">maintenance.equipment</field>
+        <field name="inherit_id" ref="maintenance.hr_equipment_view_search" />
+        <field name="arch" type="xml">
+            <xpath expr="//filter[@name='inactive']" position="after">
+                <separator/>
+                <filter string="Allow Consumptions" name="allow_consumptions" 
+                        domain="[('allow_consumptions', '=', True)]"/>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="hr_equipment_view_tree" model="ir.ui.view">
+        <field name="model">maintenance.equipment</field>
+        <field name="inherit_id" ref="maintenance.hr_equipment_view_tree" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='category_id']" position="after">
+                <field name="allow_consumptions" />
+            </xpath>
+        </field>
+    </record>
+
+    <record id="hr_equipment_view_form" model="ir.ui.view">
+        <field name="model">maintenance.equipment</field>
+        <field name="inherit_id" ref="maintenance.hr_equipment_view_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//div[@name='button_box']" position="inside">
+                <button class="oe_stat_button" type="object" name="action_view_stock_picking_ids"
+                        icon="fa-ticket"
+                        attrs="{'invisible': [('allow_consumptions', '=', False)]}"
+                        groups="stock.group_stock_user">
```

Comment:
I prefer to add the permissions on the view. In this case it seems right, but I found issues on real life due to adding the the permissions inside :thinking:

Reply:
It is not failing because you are not showing how many items, are used, but if you add the computed field, that would be a failure probably.

Reply:
No problem, I'll remember it as a good practice. Added also to `search` view